janus_core/calculations/md.py

```diff
@@ -222,6 +228,8 @@ def __init__(
         write_kwargs: OutputKwargs | None = None,
         post_process_kwargs: PostProcessKwargs | None = None,
         correlation_kwargs: list[CorrelationKwargs] | None = None,
+        plumed_input: str | PathLike | None = None,
```

Comment:
```suggestion
        plumed_input: PathLike | None = None,
```
`PathLike` is just a alias for `str | Path`